Examples/CaseStudies/SwiftUICaseStudiesTests/01-GettingStarted-BindingBasicsTests.swift

```diff
@@ -10,17 +10,17 @@ final class BindingFormTests: XCTestCase {
       BindingForm()
     }
 
-    await store.send(.set(\.sliderValue, 2)) {
+    await store.send(\.sliderValue, 2) {
```

Comment:
Because bindable actions are case pathable, we can further avoid the `.set` helper (so maybe it can even be deprecated).